cypress/e2e/general.cy.js

```diff
@@ -0,0 +1,97 @@
+const testURLRoot = "http://127.0.0.1:50722/templates/";
+
+function setIntercepts() {
+  cy.intercept(
+    {
+      method: "GET",
+      url: "/template",
+      hostname: "api-test.ramanchada.ideaconsult.net",
+    },
+    {
+      fixture: "json/bk_rcapi_template_default.json",
+    }
+  ).as("getAllTemplates");
+  cy.intercept(
+    {
+      method: "POST",
+      url: "/template",
+      hostname: "api-test.ramanchada.ideaconsult.net",
+    },
+    {
+      fixture: "json/bk_rcapi_template_default.json",
+    }
+  ).as("createNewTemplate");
+  cy.intercept(
+    {
+      method: "GET",
+      url: "/api/projects.json",
+      hostname: "enanomapper.adma.ai",
+    },
+    {
+      fixture: "json/bk_rcapi_projects_default.json",
+    }
+  ).as("getAllProjects");
+}
+
+describe("General site functionality", () => {
+
+  beforeEach(() => {
+    setIntercepts();
+    cy.visit(testURLRoot);
+  });
+
+  it("displays the under development notice", () => {
+    cy.get("#root > div > p")
+      .should("have.class", "underDev")
+      .and("be.visible");
+  });
+
+  it("can open an existing draft for editing", () => {
+    cy.get('[data-cy="draft"]').click();
+    cy.get(".nonSelected td").eq(1).click();
+    cy.get('[data-cy="Edit blueprint"]').click();
+
+    cy.get("#Save").click();
```

Comment:
@sergesoroka Why do we need to click "Save" here? It's not a bad idea to test saving, but to be meaningful, we should intercept the POST request that we'll make and check its payload for the expected JSON.